bindings/cpp/CMakeLists.txt

```diff
@@ -80,4 +80,25 @@ if(APPLE)
 endif()
 
 include(GoogleTest)
-gtest_discover_tests(opendal_cpp_test)
\ No newline at end of file
+gtest_discover_tests(opendal_cpp_test)
+
+# Documentation
+set(PROJECT_DOCUMENT_SOURCE ${CMAKE_SOURCE_DIR}/include ${CMAKE_SOURCE_DIR}/README.md)
+string(REPLACE ";" " " PROJECT_DOCUMENT_SOURCE "${PROJECT_DOCUMENT_SOURCE}")
+file(DOWNLOAD https://raw.githubusercontent.com/jothepro/doxygen-awesome-css/main/doxygen-awesome.css ${CMAKE_BINARY_DIR}/doxygen-awesome.css)
```

Comment:
Please don't download from github raw content, use CDN instead, for example: 

https://cdn.jsdelivr.net/gh/jothepro/doxygen-awesome-css@2.2.1/doxygen-awesome.min.css

Benefits:

- Pinned to version 2.2.1
- Minimize on demand.

Reply:
It looks great. updated in new commit.